### DISCUSSION TITLE
Configure metrics and limits in .github repo

### DISCUSSION THREAD

**Discussion by binford2k** [community/OSPO#25] [Dashboard]
While writing #24, I realized that it would be super useful to configure the acceptable limits for various metrics I'd like reported on. For example, maybe the default MTTR limit would be 90 days, but I'd like to configure ours to flag at 180 days or some such. Other things could be configured too. Right now, there's a static set of widgets for last contribution date, but maybe I'd like to choose other widgets to pin.

What makes most sense to me is just a .`yaml` file in our `.github` repo, like our probot config at https://github.com/puppetlabs/.github/blob/main/.github/config.yml.

**Comment by ahpook** [comment]
I think this is a bit more customization than we're looking to build for the dashboard. 

**BUT**

For this kind of advanced use case we are looking to provide a data API that will let you pull the same timeseries info we are using internally to generate the visualizations. Users have asked for the ability to combine the dashboard info with, for example, internal data on SLOs and teams.

Would that help with your use-case, so you could build those viz in your own grafana instance or the like?

**Reply by binford2k** [reply]
Oh, I like that idea a lot. Right now, we're crunching a lot of it ourselves with a *ton* of manual queries and then a lot of mathing. A data API would make that a ton easier and probably more reliable.

(and I wouldn't get myself rate limited every time I worked on the reporting pipeline 🤣)
